fossunited/stack/doctype/stack/stack.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, Frappe x FOSSUnited and contributors
+# For license information, please see license.txt
+
+# import frappe
+from frappe.model.document import Document
+
+
+class Stack(Document):
```

Comment:
shouldn't this be `StackItem` and multiple of these are pulled together to actually show the Stack in the UI?

Reply:
I think `Stack` is fine as it is 😅